packages/coingecko/src/query/index.ts

```diff
@@ -26,3 +26,40 @@ export function ping(): Ping {
   }
   throw Error(response.statusText);
 }
+
+export function coinsList(Input: Input_coinsList): Array<CoinsList> {
+  const inputs = Input.include_platform
```

Comment:
Instead of this checkout https://github.com/Niraj-Kamdar/defiwrapper/blob/nk/simple-price/packages/coingecko/src/query/index.ts and use this way to pass params

Reply:
You can cherry pick commits or stack the branch if you need any code from my branch.

Reply:
Done!

Reply:
Done #20, waiting for review